# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/VideoEngine.cpp
# Commit: f970e7758b64
# Full Hash: f970e7758b6474aaee9ac43d0fcc74f053bed0ab
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-04-21 21:12:46
# Description:
#   Bug 1828722 - Revert 30340541c9a1 - Crash in case VideoCaptureAVFoundation is destroyed unexpectedly. r=webrtc-reviewers,mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D175853
# ==============================================================================

diff -r 1455946414f1 -r f970e7758b64 dom/media/systemservices/VideoEngine.cpp
--- a/dom/media/systemservices/VideoEngine.cpp	Fri Apr 21 08:42:26 2023 +0000
+++ b/dom/media/systemservices/VideoEngine.cpp	Fri Apr 21 08:47:46 2023 +0000
@@ -104,9 +104,6 @@
 
   if (!found) {
     WithEntry(aId, [&found](CaptureEntry& cap) {
-      if (cap.mVideoCaptureModule) {
-        cap.mVideoCaptureModule->NotifyReleasing();
-      }
       cap.mVideoCaptureModule = nullptr;
       found = true;
     });